dataprofiler/profilers/categorical_column_profile.py

```diff
@@ -61,6 +68,20 @@ def __init__(self, name: str | None, options: CategoricalOptions = None) -> None
                 options.max_sample_size_to_check_stop_condition
             )
 
+            if options.cms:
+                self._heavy_hitters_threshold = options.heavy_hitters_threshold
+                self.num_hashes = datasketches.count_min_sketch.suggest_num_hashes(
+                    options.cms_confidence
+                )
+                self.num_buckets = datasketches.count_min_sketch.suggest_num_buckets(
+                    options.cms_relative_error
+                )
+                self.cms = base64.b64encode(
```

Comment:
What's the i

Reply:
if you don't encode the bytes as a string, you get "JSON can't serialize bytes" error on json.dumps()